### DISCUSSION TITLE
What does the hideTab do?

### DISCUSSION THREAD

**Discussion by Ironbatpug** [apache/superset#23815] [Q&A / Help]
Hello
I have implemented the dashboard embedding logic, and everything looks good, but I did not find any difference between the hidetab true and false. I did not find any documentation about this, so could you please tell what does it do?

**Comment by Ironbatpug** [comment]
In this picture the hidetab, hidechartcontrols and hidetitle are true: ![image](https://user-images.githubusercontent.com/48181291/234776864-0ed9a3bb-fccb-4751-b329-aaa849fcd54f.png)

And here all of them are false:
![image](https://user-images.githubusercontent.com/48181291/234777103-e9f80c98-ed25-4e55-9849-74dbfb675ab5.png)

**Comment by myonster** [comment]
I was wondering too because I wanted to hide the 'Overview' tab. Were you able to come to an answer in the end?
